azurerm/internal/services/compute/windows_virtual_machine_resource.go

```diff
@@ -175,7 +175,6 @@ func resourceWindowsVirtualMachine() *schema.Resource {
 			"license_type": {
 				Type:     schema.TypeString,
 				Optional: true,
-				ForceNew: true,
 				ValidateFunc: validation.StringInSlice([]string{
```

Comment:
running the tests for this unfortunately `None` causes the tests to fail - so we'll need a DiffSuppressFunc to capture the fact that `None` is exposed as both "None" and ""

```

				DiffSuppressFunc: func(_, old, new string, _ *schema.ResourceData) bool {
					if old == "None" && new == "" || old == "" && new == "None" {
						return true
					}

					return false
				},
```

I hope you don't mind but I'll push a commit to fix this